### PR TITLE
Clarify Stored Requirements

### PR DESCRIPTION
**Pull request by oconnetf**
This is intended to resolve https://github.com/adlnet/xAPI-Spec/issues/931.

### CODE REVIEW

File: xAPI-Data.md [hunk 1]
```diff
@@ -1571,7 +1571,7 @@ Used to record the time at which the experience described in the Statement.
 ###### Requirements
 
 * For requirements pertaining to the Timestamp data type, see [ISO 8601 Timestamps](#timestamps) below.
-* The stored property MUST be set by the LRS; An LRS MUST validate and then overwrite any value currently in the 
+* The stored property MUST be set by the LRS; An LRS SHOULD validate and then overwrite any value currently in the
```
Comment (DavidTPate) [inline xAPI-Data.md]: I think there just needs to be a slight change here. Based on the call on Wednesday I understood this to be `An LRS SHOULD validate and then MUST overwrite any value currently in the stored property of a Statement it received`

The key part being the `MUST` overwrite, as right now it reads as a `SHOULD` for overwriting.

Comment (oconnetf) [inline xAPI-Data.md]: Hmm. On closer reading, I think that's probably right.

**Comment by garemoko** [comment]
+1

I don't think the MUST is necessary in "and then MUST overwrite any value currently..." because of the MUST in the previous sentence, but equally I don't think it hurts.

**Comment by bscSCORM** [comment]
👍 , and I think the "then MUST overwrite" is necessary because otherwise one could argue about whether or not using the value that was passed in _or_ writing a new value only when no value is passed in satisfies the first MUST

**Pull request merged**